docs/products/alloydbomni/reference/list-of-extensions.md

```diff
@@ -0,0 +1,189 @@
+    Cryptographic functions
+-   [unaccent](https://www.postgresql.org/docs/current/unaccent.html):
+    Text search dictionary that removes accents
+
+### Auditing
+
+-   [tcn](https://www.postgresql.org/docs/current/tcn.html): Triggered
+    change notifications
+
+### Geographical features
+
+-   [earthdistance](https://www.postgresql.org/docs/current/earthdistance.html):
+    Calculate great-circle distances on the surface of the Earth
+
+### Procedural language
+
+- [plpgsql](https://www.postgresql.org/docs/current/plpgsql.html):
+  PL/pgSQL procedural language
+- [plproxy](https://plproxy.github.io/): Procedural language handler that allows remote
+  procedure calls among PostgreSQL databases
+- [plv8](https://plv8.github.io/): Provides a procedural language for enabling JavaScript
+
+### Connectivity
+
+-   [dblink](https://www.postgresql.org/docs/current/contrib-dblink-function.html):
+    Connect to other AlloyDB Omni databases from within a database
+-   [postgres_fdw](https://www.postgresql.org/docs/current/postgres-fdw.html):
+    Foreign-data wrapper for remote AlloyDB Omni servers
+
+### Utilities
+
+- [amcheck](https://www.postgresql.org/docs/current/amcheck.html): Tools to verify table
+  and index consistency
+- [auto_explain](https://www.postgresql.org/docs/current/auto-explain.html): Log execution
+  plans of slow queries
+- [autoinc](https://www.postgresql.org/docs/17/contrib-spi.html#CONTRIB-SPI-AUTOINC):
+  Function for autoincrementing fields that stores the next value of a sequence into an
+  integer field
+- [google_columnar_engine](https://cloud.google.com/alloydb/docs/reference/columnar-engine-flags):
+  Adds support for
+  [the columnar engine](https://cloud.google.com/alloydb/docs/columnar-engine/about)
+- [google_db_advisor](https://cloud.google.com/alloydb/docs/use-index-advisor): Provides
+  the index advisor, which recommends indexes to speed up query processing
+- [google_ml_integration](https://cloud.google.com/alloydb/docs/ai/invoke-predictions):
+  Allows interacting with Vertex AI, registering AI model endpoints, generating embeddings,
+  and invoking predictions within your SQL environment
+- [hypopg](https://github.com/HypoPG/hypopg): Adds support for hypothetical (virtual)
+  indexes
+- [insert_username](https://www.postgresql.org/docs/current/contrib-spi.html#CONTRIB-SPI-INSERT-USERNAME):
+  Functions for tracking who changed a table
+- [intagg](https://www.postgresql.org/docs/current/intagg.html).
+  Integer aggregator and enumerator (obsolete).
+- [intarray](https://www.postgresql.org/docs/current/intarray.html):
+  Functions, operators, and index support for 1-D arrays of integers
+- [lo](https://www.postgresql.org/docs/current/lo.html): Large Object
+  maintenance
+- [moddatetime](https://www.postgresql.org/docs/current/contrib-spi.html#CONTRIB-SPI-MODDATETIME):
+  Functions for tracking last modification time
+- [pageinspect](https://www.postgresql.org/docs/current/pageinspect.html):
```

Comment:
Some of those have been removed by us, see ticket here for the list: 
https://github.com/aiven/aiven-core/pull/51357